packages/bot-avs/src/main.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable no-console */
```

Comment:
Remove this ESLint setting and use `console.info`. Please also add a copyright preamble (see other code in wire-web-packages) at the beginning of every file.

Reply:
https://github.com/wireapp/wire-web-packages/pull/3070/commits/7d09467b0224d03da757b9ee3fcc61452d293db2